api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the seccomp-operator v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=seccomp-operator.k8s-sigs.io
```

Comment:
Need input on what the group name for this resource should be. `seccomp-operator.sigs.k8s.io` would make sense but trying to use it produces this error:

```
The CustomResourceDefinition "seccompprofiles.seccomp-operator.sigs.k8s.io" is invalid: metadata.annotations[api-approved.kubernetes.io]: Required value: protected groups must have approval annotation "api-approved.kubernetes.io", see https://github.com/kubernetes/enhancements/pull/1111
```

Reply:
I'm not sure if we would get approval if we request it from the API reviewers. :thinking:  Otherwise we don't have other options to use a pseudo name, right?

Reply:
Tricky one! Would you be opposed to use something internal (without k8s-sigs.io) and open an issue for us to resolve this later on depending on whether or not the project gets renamed? I'd rather we go for a name submission/approval only once if possible.

We probably could start with `seccompprofiles.seccomp-operator.io` and then change and submit for approval when/if we finally decide on the project renaming.

Reply:
Got a response in the [slack message where I raised the question](https://kubernetes.slack.com/archives/C5P3FE08M/p1600968242008600)

So I guess `x-k8s.io` is the namespace we have to choose.

Xref: https://github.com/kubernetes-sigs/cluster-api/issues/1073#issuecomment-514265501

Reply:
Done https://github.com/kubernetes-sigs/seccomp-operator/issues/135

Reply:
Fixed